app/src/main/java/com/sdpteam/connectout/profileList/ProfileFilterFragment.java

```diff
@@ -0,0 +1,75 @@
+package com.sdpteam.connectout.profileList;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.ToggleButton;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import com.sdpteam.connectout.R;
+
+public class ProfileFilterFragment extends Fragment {
```

Comment:
Please provide documentation for this class so that we can quickly understand what it does if we want to use it.